### PR TITLE
Remove the trashed state from fileId endpoint

### PR DESCRIPTION
**Pull request by SwikritiT**
Related workpackage: https://community.openproject.org/projects/nextcloud-integration/work_packages/52045

### CODE REVIEW

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 1]
```diff
@@ -214,7 +190,6 @@ Feature: retrieve file information of a single file, using the file ID
       "owner_name",
       "modifier_id",
       "modifier_name",
-      "trashed",
```
Comment (individual-it) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:217]: let's put that in 
```
"not": {
      "required": [
```
to make sure the parameter is really gone.
Similar for the other tests

File: tests/acceptance/features/api/getFileinfoByFileIDAPI.feature [hunk 2]
```diff
@@ -164,26 +153,13 @@ Feature: retrieve file information of a single file, using the file ID
...
```
Comment (individual-it) [inline tests/acceptance/features/api/getFileinfoByFileIDAPI.feature:167]: let's leave that line here, as its in not-required, that will make sure its really deleted

**Review by individual-it — CHANGES_REQUESTED** [review]
looks very good, just tiny suggestions for the acceptance tests

**Review by Kharonus — APPROVED** [review]
LGTM.

From what I can tell, the functionality change is exactly as we discussed, hence accepted from our side.

**Review by individual-it — APPROVED** [review]

**Pull request merged**